actions/push-to-gcs/action.yaml

```diff
@@ -0,0 +1,95 @@
+name: Push to cloud storage
+description: Composite action to push to Google Cloud Storage
+inputs:
+  bucket:
+    description: |
+      Name of bucket to upload to. Will default to grafanalabs-${repository.id}-${environment}
+    default: ""
+  path:
+    description: |
+      Path to the object(s) to upload. Can either include a filename to upload 1 file,
+      or can be used in conjunction with the `glob` option to upload files from a path.
+      Valid examples include `thing.txt` and `path/to/thing.txt`.
+      Valid examples when also using `glob` include `path/to`.
+    required: true
+  bucket_path:
+    description: |
+      The path in the bucket to save the object(s). Defaults to the root of the bucket.
+    default: ""
+  environment:
+    description: |
+      Environment for uploading objects (can be either dev or prod).
+    default: dev
+  glob:
```

Comment:
I like this `glob` thing - I wonder if it would make sense to make it part of the `path` so it's more clear to everybody?
Like check if there are `*` or `**` and have a step in this github action which will either resolve to a single object or a glob pattern? No strong opinions, I think it will just make things a little bit easier. Feel free to ignore if you have already thought about it and it works better your way!

Reply:
I took a stab at implementing your recommendation, and I found it to be more confusing than leaving them separate. It did however give me the opportunity to revisit and simplify by eliminating the logic I was applying to those inputs, so now we simply pass the `path` and `glob` inputs directly to the downstream action. 

In the README I documented several cases for how to use these inputs in conjunction with each other. Let me know what you think!